src/lib.rs

```diff
@@ -1086,6 +1110,13 @@ pub struct BindgenOptions {
 
     /// Whether to prepend the enum name to bitfield or constant variants.
     pub prepend_enum_name: bool,
+
+    /// Whether rustfmt should format the generated bindings.
+    pub format_bindings: bool,
```

Comment:
Super nitpicky (sorry!), but I'd prefer `rustfmt_bindings` and `--rustfmt-bindings` (and similar for the config files).